cli/src/commands/git.rs

```diff
@@ -751,7 +751,7 @@ fn do_git_clone(
             unreachable!("we didn't provide any globs")
         }
     })?;
-    print_git_import_stats(ui, &stats.import_stats)?;
+    print_git_import_stats(ui, &stats.import_stats, false)?;
```

Comment:
nit: might be better to print the imported refs on `jj git clone`/`import`, but that can be decided later.

Reply:
I wasn't sure what to do with local branches, so skipped it for now. Can turn it on if you think it makes sense

Reply:
resolved

Reply:
`jj git clone git@github.com:martinvonz/jj.git  --colocate` 
now gives
```
Fetching into new repo in "/tmp/tmp.nCGWca6Ajb/jj"
branch: CacheMyWorkingCopy@origin        [new] untracked
branch: arxanas/ignore@origin            [new] untracked
branch: aseipp/push-mupwvrwmxuvm@origin  [new] untracked
branch: aseipp/push-mwwotvxyruwp@origin  [new] untracked
branch: aseipp/push-oolmkoukrxxt@origin  [new] untracked
branch: aseipp/push-srtxpxttmqlr@origin  [new] untracked
branch: aseipp/push-tuxwuxlpsttr@origin  [new] untracked
branch: aseipp/push-uytvkxyqyspn@origin  [new] untracked
branch: aseipp/push-zkyqxypkvulk@origin  [new] untracked
branch: gh-pages@origin                  [new] untracked
branch: ig/contributing@origin           [new] untracked
branch: ig/faq-move@origin               [new] untracked
branch: ig/retrytest@origin              [new] untracked
branch: ig/tmp-retrytest-with-testfail@origin [new] untracked
branch: jt/ignore@origin                 [new] untracked
branch: main@origin                      [new] untracked
branch: missingop@origin                 [new] untracked
branch: necauqua/signing@origin          [new] untracked
branch: pager@origin                     [new] untracked
branch: prev-next@origin                 [new] untracked
branch: push-5640745a446c47efb1bd819dd4c95a66@origin [new] untracked
branch: push-numyszkprmty@origin         [new] untracked
branch: push-onymwnywputl@origin         [new] untracked
branch: push-ppomxyyutuqm@origin         [new] untracked
branch: push-pvwusmnnpwul@origin         [new] untracked
branch: push-qvrktsnwpkzu@origin         [new] untracked
branch: push-rmzwokuwmwtx@origin         [new] untracked
branch: push-ruzwxounnsut@origin         [new] untracked
branch: push-rvuprqxytrtu@origin         [new] untracked
branch: push-skzwxzopxszv@origin         [new] untracked
branch: push-slnqmxmxyuvr@origin         [new] untracked
branch: push-sovruzpmulyr@origin         [new] untracked
branch: push-sxvnsknzvxku@origin         [new] untracked
branch: push-tlpqtmtonpqp@origin         [new] untracked
branch: push-ytvunlzsxzvx@origin         [new] untracked
branch: push-zmyoqzynpltt@origin         [new] untracked
branch: talp/moqlklnlrywq@origin         [new] untracked
branch: talp/oqpyxkqkuyvm@origin         [new] untracked
branch: testing@origin                   [new] untracked
tag: v0.10.0                             [new]
tag: v0.11.0                             [new]
tag: v0.12.0                             [new]
tag: v0.13.0                             [new]
tag: v0.14.0                             [new]
tag: v0.3.0                              [new]
tag: v0.3.1                              [new]
tag: v0.3.2                              [new]
tag: v0.3.3                              [new]
tag: v0.4.0                              [new]
tag: v0.5.0                              [new]
tag: v0.5.1                              [new]
tag: v0.6.0                              [new]
tag: v0.6.1                              [new]
tag: v0.7.0                              [new]
tag: v0.8.0                              [new]
tag: v0.9.0                              [new]
Working copy now at: uwlkpvyw c8c91829 (empty) (no description set)
Parent commit      : npnmltou 89cf6a00 main | graphlog: refactor graphlog::Edge enum and its usage
Added 334 files, modified 0 files, removed 0 files
```

Reply:
Oops, this looks verbose than I expected. Maybe better for `jj git clone` to not show ref changes? What do you think?